clang/test/dpct/python_setup_migration/case_001/case_001_framework.cpp

```diff
@@ -0,0 +1,8 @@
+// RUN: rm -rf %T && mkdir -p %T
+// RUN: cd %T
+// RUN: cp %S/python_setup_input.py ./python_setup_input.py
+// RUN: mkdir -p subdir
+// RUN: dpct -in-root ./ -out-root out  ./python_setup_input.py --migrate-build-script-only
```

Comment:
If the in-root directory contains both *.py files and CMake script files, SYCLomatic will migrate these two kinds of files together when "--migrate-build-script-only" is specified?

If users only want to migrate one kind of  build script file such as *pl,  the file type should be also specified by "--migrate-build-script=CMake or setuppy".

Reply:
1. Yes, "--migrate-build-script-only" will migrate both CMake and Python build script files
2. Yes, the new option to migrate only Python build script files is "--migrate-build-script=Python_Setup"

Reply:
For 1, pls add a lit test for this scenario: If in-root, out-root, and migrate-build-script-only are specified, then both CMake scripts and Python scripts in the in-root directory should be migrated.
For 2:  so, when "--migrate-build-script=Python_Setup" is specified, then the Python script should be migrated together with the source migration, so pls add a lit test to verify that.

Reply:
Added the test cases